test/functional/tests/cli/test_cli_standby.py

```diff
@@ -459,3 +459,52 @@ def test_activate_neg_cache_line_size():
                     f'Expected Cache state: "{CacheStatus.standby.value}" '
                     f'Got "{cache_status.value}" instead.'
                 )
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_standby_start_stop(cache_line_size):
+    """
+        title: Start and stop a standby cache instance.
+        description: Test if cache can be started in standby state and stopped without activation.
+        pass_criteria:
+          - A cache exported object appears after starting a cache in standby state
+          - The data written to the cache exported object committed on the underlying cache device
+          - The cache exported object disappears after stopping the standby cache instance
+    """
+    with TestRun.step("Prepare a cache device"):
+        cache_size = Size(500, Unit.MebiByte)
+        cache_device = TestRun.disks['cache']
+        cache_device.create_partitions([cache_size])
+        cache_device = cache_device.partitions[0]
+
+    with TestRun.step("Start standby cache with the tested cache line size"):
+        cache_id = 1
+        cls = int(cache_line_size.value.value / Unit.KibiByte.value)
+        cache = casadm.standby_init(cache_device, cache_id, cache_line_size=cls, force=True)
+
+    with TestRun.step("Verify that a cache exported object appeared in the system"):
+        output = TestRun.executor.run_expect_success(f"ls -la /dev/ | grep 'cas-cache-{cache_id}'")
+        if output.stdout[0] != "b":
+            TestRun.fail("The cache exported object is not a block device")
+
+    with TestRun.step("Sequentially write data to the cache exported"
+                      "object (with offset set to the middle of the device)"):
+        block_size = Size(4, Unit.MebiByte)
+        offset = int(cache_size/block_size/2)
+        Dd().output(f"/dev/cas-cache-{cache_id}") \
+            .input("/dev/urandom") \
+            .block_size(block_size) \
+            .seek(offset)
+
+    with TestRun.step("Count and save a checksum for the written data on the exported object"):
+        cache_md5 = TestRun.executor.run(f"md5sum /dev/cas-cache-{cache_id}").stdout.split()[0]
```

Comment:
You should calculate md5 only for the part of cache exported object which you populated with data. The current implementation may fail because of the differences in metadata section after the standby instance is stopped